File: photomap-core-api/api-gateway-kt/src/main/java/ua/in/photomap/apigateway/controller/CsrfController.java
package ua.in.photomap.apigateway.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

@RestController
public class CsrfController {

    @GetMapping("/api/csrf")
    public Map<String, String> csrf() {
        return Map.of("result", "csrf");
    }
}

File: photomap-core-api/api-gateway-kt/src/main/kotlin/ua/in/photomap/apigateway/controller/CsrfController.kt
package ua.`in`.photomap.apigateway.controller

import org.springframework.security.web.server.csrf.CsrfToken
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import reactor.core.publisher.Mono

@RestController
class CsrfController {
    @GetMapping("/api/csrf")
    fun csrf(): Mono<Map<String, String>> =
        Mono.just(mapOf("result" to "csrf"))
} 